packages/ui/src/Slider.tsx

```diff
@@ -57,6 +58,29 @@ function isSingleValueGuard(value: SliderValue, onChange?: SingleValueChangeFn |
   return !Array.isArray(value)
 }
 
+function triggerNativeInputChange(value: string, inputEl: HTMLInputElement) {
```

Comment:
I would move it to helpers and make it a generic function to fire an event on any element

Reply:
Moved to helpers. But now we need to deploy new helpers version in order to use it in ui. So let's merge this one and then I will create another PR